.github/workflows/tag.yml

```diff
@@ -0,0 +1,28 @@
+name: flask-rebar Pull Request Tests
+
+on:
+  release:
+    types:
+    - created
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
```

Comment:
Why not 3.8?

Reply:
It was just the one that was used in travis, and it doesn't matter, since the wheel will be universal and pythoon agnostic.